elyra/pipeline/processor.py

```diff
@@ -87,12 +87,22 @@ async def export(self, pipeline, pipeline_export_format, pipeline_export_path, o
         return res
 
 
-class PipelineProcessorResponse(object):
-    def __init__(self, run_url='', object_storage_url='', object_storage_path=''):
+class PipelineProcessorResponse(ABC):
+
+    _type = None
+
+    def __init__(self, run_url, object_storage_url, object_storage_path, **kwargs):
+
+        super(PipelineProcessorResponse, self).__init__(**kwargs)
```

Comment:
I recently learned the `super` calls can be simplified.
```suggestion
        super().__init__(**kwargs)
```
Hmm, do we need the keyword args entry?  Also, I'm not sure we need the call to super for the ABC actually.

Did you find the **kwargs was necessary?  I see you define the known parameters are regular args (*args), yet call the constructor using keyword-args, so I wonder if it's confused.

Reply:
simplified, removed call to super and kwargs